src/main/java/ui/components/FilterTextField.java

```diff
@@ -242,6 +215,30 @@ private void confirmEdit() {
         positionCaret(caretPosition);
     }
 
+    // SuggestionMenu 
+    
+    private final SuggestionMenu setupSuggestion() {
```

Comment:
This method is private, so it doesn't have to be final -- it can't be overridden anyway.